pyrms/rms.py

```diff
@@ -23,6 +23,28 @@ def pygetfluxdiagram(bsol,t,centralspecieslist=[],superimpose=False,
 
 ReactionMechanismSimulator.getfluxdiagram = pygetfluxdiagram
 
+# These functions force the returned object to be a jlwrap object, 
+# to avoid the solution object gets converted into list of lists
+_threadedsensitivities_inds = Main.pyfunctionret(ReactionMechanismSimulator.threadedsensitivities, Main.Any, Main.PyAny, Main.PyAny)
+_threadedsensitivities = Main.pyfunctionret(ReactionMechanismSimulator.threadedsensitivities, Main.Any, Main.PyAny)
+# Allow us the get the solution object in the julia wrapped object without
+# it being converted into list of lists
+get = Main.pyfunctionret(Main.get, Main.Any, Main.PyAny, Main.PyAny, Main.PyAny)
+def pythreadedsensitivities(react, inds=None):
```

Comment:
Can these be fed the keyword arguments the original function can?

Reply:
This needs https://github.com/ReactionMechanismGenerator/ReactionMechanismSimulator.jl/pull/212 to be merged first